lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserSwift.cpp

```diff
@@ -297,3 +299,199 @@ lldb_private::CompilerDeclContext
 DWARFASTParserSwift::GetDeclContextContainingUIDFromDWARF(const DWARFDIE &die) {
   return CompilerDeclContext();
 }
+
```

Comment:
Should we move the -gdwarf-types support into a separate file to avoid confusion?

Reply:
Ok I separated that into `DWARFASTParserSwiftDescriptorFinder.cpp`, where I can add all of the functionality related to implementing `DWARFASTParserSwift`'s implementation of the `DescriptorFinder` interface.